### DISCUSSION TITLE
If possible, add a "exec this" or "exec that" on theme switching

### DISCUSSION THREAD

**Discussion by CrazyThunderbird** [AutoDarkMode/Windows-Auto-Night-Mode#505] [General]
Hi folks,
I try to describe it as good as i can, could there be the option to execute a file when switching to "dark mode" and another file when switching to light mode?

f.e. i control my monitors with DDC (ControlMyMonitor) For a "calibrated" night mode, so that all screens sitting on the same brightness and same color temperature.

So it would be neat if you could add a [EXEC] "Whateveryourfilename.is" when switching

For example i have 2 Bat Files
1. D65.bat (6500k color Temperature on 140cd/m² brightness) Daytime use
2. D30.bat (3000K color Temperature on 40cd/m² brightness) for nighttime

Example of my D65.bat
```
@echo off
F:\Systemtools\MonitorCFG\ControlMyMonitor.exe /LoadConfig "F:\Systemtools\MonitorCFG\M32U-D65.cfg" \\.\DISPLAY1\Monitor0 /LoadConfig "F:\Systemtools\MonitorCFG\BenQ-D65.cfg" \\.\DISPLAY2\Monitor0 /LoadConfig "F:\Systemtools\MonitorCFG\ASUS-D65.cfg" \\.\DISPLAY3\Monitor0
```

Would be pretty neat to get that automated aswell :)

**Comment by Spiritreader** [comment]
This is already available, please check out this documentation
https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/wiki/How-to-add-custom-scripts
